utils/oscap-info.c

```diff
@@ -776,19 +790,25 @@ bool getopt_info(int argc, char **argv, struct oscap_action *action)
 	while ((c = getopt_long(argc, argv, "o:i:p:", long_options, NULL)) != -1) {
```

Comment:
Do we still accept short options?

Reply:
Do you mean here in oscap info module? The oscap info doesn't have any short options so I think we don't accept them.

Reply:
Should the `"o:i:p:"` here be removed?